### PR TITLE
Info tools for gcloud node setup

### PR DESCRIPTION
**Pull request by posvyatokum**
Two scripts to display details of a node setup. 
One shows "canonical" config.json hash.
Another shows a result of "./neard -V". 

I'm not an expert, but it works on my machine (tm).

### CODE REVIEW

File: scripts/remote_diff/utils.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        shell=True,
+    )
+    return zone_call.stdout.replace('\n', '')
+
+
+def run_on_machine(command, user, host, project):
+    zone = get_zone(project, host)
+    call = subprocess.run(
+        [
+            "gcloud compute ssh {}@{} --command='{}' --project {} --zone {} ".
```
Comment (nikurt) [inline scripts/remote_diff/utils.py:23]: Maybe use f-strings to make it easier for the readers?

```suggestion
            f"gcloud compute ssh {user}@{host} --command '{command}' --project {project} --zone {zone}"
```

File: scripts/remote_diff/show_config_hash.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import sys
```
Comment (nikurt) [inline scripts/remote_diff/show_config_hash.py:1]: Please add a command explaining the purpose of this script and an example of its intended use.

File: scripts/remote_diff/show_neard_version.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import sys
```
Comment (nikurt) [inline scripts/remote_diff/show_neard_version.py:1]: Please add a command explaining the purpose of this script and an example of its intended use.

File: scripts/remote_diff/utils.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import subprocess
+
+
+def get_zone(project, host):
+    zone_call = subprocess.run(
+        [
+            "gcloud compute instances list \
+        --project={} | grep ^{} | awk -F ' ' '{{print $2}}'".format(
```
Comment (nikurt) [inline scripts/remote_diff/utils.py:8]: Please consider f-strings.

**Comment by posvyatokum** [comment]
@nikurt @jakmeier I think I resolved everything, can one of you please approve? :upside_down_face:

**Review by nikurt — APPROVED** [review]

**Pull request merged**